temporal-serviceclient/src/main/java/io/temporal/serviceclient/WorkflowServiceStubsImpl.java

```diff
@@ -151,6 +164,23 @@ public WorkflowServiceStubsImpl(
     log.info(String.format("Created GRPC client for channel: %s", channel));
   }
 
+  private ScheduledExecutorService startConnectionBackoffResetter(Duration backoffResetFrequency) {
+    ScheduledExecutorService executor = Executors.newSingleThreadScheduledExecutor();
```

Comment:
nit: 
return Executors.new....

Reply:
We submit a task below, so this will not be possible...